angular-portfolio/backend/src/main/java/com/google/sps/servlets/AwardsServlet.java

```diff
@@ -83,4 +67,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("application/json;");
     response.getWriter().println(gson.toJson(awards));
   }
+
+  private Entity createNewEntity(Entity awardEntity, string key, long year, string image, string description) {
```

Comment:
Does this work? String in Java is a class, not a primitive so it should be "String" instead of "string"